gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.8-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.10-bin.zip
```

Comment:
there's not a meaningful amount of change, still: https://docs.gradle.org/current/userguide/test_kit.html